### DISCUSSION TITLE
Add more support to pkgconfig

### DISCUSSION THREAD

**Discussion by r2d2Proton** [microsoft/vcpkg#17450] [Ideas]
Will PkgConfig be supported?
Is there any technical reason why it cannot or will not be supported?

I am looking at this now but any help is appreciated.

Thank you.

**Comment by dg0yt** [comment]
What kind of support do you ask for?
There is:
- `vcpkg_find_acquire_program(PKGCONFIG)` (but PKGCONFIG missing in documentation)
- `vcpkg_fixup_pkgconfig(...)`

**Reply by dg0yt** [reply]
`vcpkg_find_acquire_program(PKGCONFIG)` is also used by `vcpkg_configure_make(...)`.

**Comment by dg0yt** [comment]
Pkgconfig support for manifest mode is just about to be created. Cf. https://github.com/microsoft/vcpkg/issues/17481

**Comment by JackBoosY** [comment]
We should support automatic parsing of pkgconfig files and printing usage in vcpkg like cmake.
